vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/verdict.rb

```diff
@@ -0,0 +1,26 @@
+module SecureRooms
+
+  module AccessRules
+
+    class Verdict
+
+      attr_accessor :reason, :result_code
+
+      def initialize(result_code, reason = nil)
+        @result_code = result_code
+        @reason = reason
+      end
+
+      def pass?
+        @result_code == :pass
+      end
+
+      def has_result_code?(code)
```

Comment:
Ah, ok. I did a find on this method and didn't see it anywhere (because it's `have`). I'm cool with keeping it.